lighthouse/classes/event_properties/definitions/samples_with_cog_uk_id.py

```diff
@@ -40,8 +40,8 @@ def errors(self) -> List[str]:
     def value(self):
         with self.retrieval_scope():
             samples = list(self._samples_from_destination.value.values())
-            add_cog_barcodes_from_different_centres(samples)
-            update_mlwh_with_cog_uk_ids(samples)
+            updated_samples = add_cog_barcodes_from_different_centres(samples)
+            update_mlwh_with_cog_uk_ids(updated_samples)
```

Comment:
I would add a comment before line 44, so in TODO remove update_mlwh_with_cog_uk_ids when the transition has happened.  When we have moved completely to queues we want to not modify anymore lighthouse_sample